tensorrt_llm/bench/benchmark/low_latency.py

```diff
@@ -45,10 +45,11 @@
     default=None,
     help="Path to a serialized TRT-LLM engine.",
 )
-@optgroup.option("--backend",
-                 type=click.Choice(ALL_SUPPORTED_BACKENDS),
-                 default="pytorch",
-                 help="The backend to use when running benchmarking.")
+@optgroup.option(
+    "--backend",
+    type=click.Choice(ALL_SUPPORTED_BACKENDS),
```

Comment:
It is worth updating `ALL_SUPPORTED_BACKENDS` to be pulled directly from the new enum?

```
ALL_SUPPORTED_BACKENDS = [backend.canonical_value for backend in BackendType]
```

Reply:
I've unified here with BackendType.